src/Redis__Fast.xs

```diff
@@ -666,6 +666,12 @@ static redis_fast_reply_t  Redis__Fast_run_cmd(Redis__Fast self, int collect_err
         // else destructor will release cbt
 
         if(res == WAIT_FOR_EVENT_READ_TIMEOUT || res == WAIT_FOR_EVENT_WRITE_TIMEOUT) {
+            if(self->flags & (FLAG_INSIDE_TRANSACTION | FLAG_INSIDE_WATCH)) {
+                self->is_connected = 0;
+                self->ac->replies.head = NULL;
```

Comment:
we should not touch `self->ac->replies`.
to clean up `self->ac->replies`, see the implementation of quit. https://github.com/shogo82148/Redis-Fast/blob/master/src/Redis__Fast.xs#L1113-L1120